dags.yaml

```diff
@@ -1066,6 +1066,28 @@ bqetl_adjust:
     - impact/tier_2
     - repo/bigquery-etl
 
+
+bqetl_adjust_derived:
+  default_args:
+    depends_on_past: false
+    email:
+      - telemetry-alerts@mozilla.com
+      - mhirose@mozilla.com
+    email_on_failure: true
+    email_on_retry: true
```

Comment:
I'd generally suggest not having `email_on_retry`, it created noise and it's very unlikely anyone would perform any action at that point so it's mostly just spamming your mailbox.